lab1.ipynb

```diff
@@ -2648,15 +2698,22 @@
     "        if m.group(4) == 'G':\n",
     "            throughputs.append(float(m.group(3)))\n",
     "        elif m.group(4) == 'M':\n",
-    "            throughputs.append(float(m.group(3))/1000)\n",
+    "            throughputs.append(float(m.group(3)) / 1000)\n",
     "    return sum(throughputs) / len(throughputs)\n",
     "\n",
-    "print('Throughput test 1: %.2f Gbps\\nSimulation time: %.2fs\\n' %\n",
-    "      (parse_iperf_throughput(output, client),\n",
-    "      output.end_time - output.start_time))\n",
-    "print('Throughput test 2: %.2f Gbps\\nSimulation time: %.2f' %\n",
-    "      (parse_iperf_throughput(output_2, client),\n",
-    "      output_2.end_time - output_2.start_time))\n"
+    "\n",
+    "print(\n",
+    "    'Throughput test 1: %.2f Gbps\\nSimulation time: %.2fs\\n' % (\n",
+    "        parse_iperf_throughput(output,\n",
+    "                               client), output.end_time - output.start_time\n",
```

Comment:
Just curious, is this what the code style explicitly suggests? Or would it be consistent with the style and more readable to break after `),` and indent to the same level as `parse` in the line above?

Reply:
The interesting section from the style guide is this one: https://google.github.io/styleguide/pyguide.html#34-indentation
Basically, we can always use either hanging indent or indent with vertical alignment. Here, both were mixed, which saves some lines but I agree, readability using your suggestion is better. I am going to look into yapf's options again and then change the formatting here.

Reply:
Quickly compared to pep8 and it seems to suggest the same.